src/components/TopBar.js

```diff
@@ -187,16 +186,14 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
               style: { flex: 'none', display: 'flex', justifyContent: 'center', alignItems: 'center', height: 95 },
               role: 'listitem'
             }, [
-              isDatastage() || isBioDataCatalyst() ?
-                h(Clickable, {
-                  href: Nav.getLink('workspaces'),
-                  style: {
-                    backgroundColor: 'white', fontSize: 18, fontWeight: 500, color: colors.accent(),
-                    borderRadius: 5, boxShadow: '0 2px 4px 0 rgba(0,0,0,.25)',
-                    width: 250, height: 56, display: 'flex', alignItems: 'center', justifyContent: 'center'
-                  }
-                }, ['SIGN IN']) :
-                h(SignInButton)
+              h(Clickable, {
+                ...(isDatastage() || isBioDataCatalyst()) ? { href: Nav.getLink('workspaces') } : { onClick: signIn },
```

Comment:
I'm not sure either honestly but that behavior predated my change, so I kept it.

Reply:
Asking around, the folklore is that it is necessary to redirect to the page because BioData Catalyst requires to have the disclaimer text visible when you choose to log in (and Datastage is actually just an older name of BioData Catalyst).